google-cloud-pubsub/lib/google/cloud/pubsub/policy.rb

```diff
@@ -162,10 +163,9 @@ def to_grpc
           Google::Iam::V1::Policy.new(
             etag:     etag,
             bindings: roles.keys.map do |role_name|
-              next if roles[role_name].empty?
               Google::Iam::V1::Binding.new(
                 role:    role_name,
-                members: roles[role_name]
+                members: roles[role_name].compact
```

Comment:
I think this changes the logic. We don't expect `roles[role_name]` to contain nils, but we want to drop the entire binding if it is empty.

Reply:
Fixed, so that the change is now exactly the same as in `Policy` in #10577.